ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/qos-interfaces.j2

```diff
@@ -19,12 +19,12 @@
 
 | Interface | Trust | Default DSCP | Default COS | Shape rate |
 | --------- | ----- | ------------ | ----------- | ---------- |
-{%     for ethernet_interface in ethernet_interfaces_qos %}
-{%         set qos_trust = ethernet_interfaces[ethernet_interface].qos.trust | arista.avd.default('-') %}
-{%         set qos_dscp = ethernet_interfaces[ethernet_interface].qos.dscp | arista.avd.default('-') %}
-{%         set qos_cos = ethernet_interfaces[ethernet_interface].qos.cos | arista.avd.default('-') %}
-{%         set shape_rate = ethernet_interfaces[ethernet_interface].shape.rate | arista.avd.default('-') %}
-| {{ ethernet_interface }} | {{ qos_trust }} | {{ qos_dscp }} | {{ qos_cos }} | {{ shape_rate }} |
+{%     for ethernet_interface in ethernet_interfaces_qos | arista.avd.convert_dicts('name') | arista.avd.natural_sort('name') %}
```

Comment:
the conversion and sorting is not needed here, since you build the `ethernet_interfaces_qos` list above in line 2-15. With these changes the `ethernet_interfaces_qos` goes from being a list of keys to a list of dicts.